llama_index/text_splitter/utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Callable, List
+
+from llama_index.text_splitter.types import TextSplitter
+
+
```

Comment:
Is there an advantage to having these as utils vs. functions on base splitter class? Do these get used without/outside of splitters?

Reply:
I prefer to share code via standalone functions, rather than through inheritance

Reply:
Fair!